src/commands/dev/dev.mjs

```diff
@@ -494,10 +505,20 @@ const dev = async (options, command) => {
     startPollingForAPIAuthentication({ api, command, config, site, siteInfo })
   }
 
+  log(`${NETLIFYDEVWARN} Setting up local development server`)
+
+  const { configPath: configPathOverride } = await runBuild({ cachedConfig, options, settings, site })
+
+  // When using the `serve` flag, we want to use the production functions built
+  // by Netlify Build rather than building them from source.
+  const dist = Boolean(options.serve)
```

Comment:
Is it possible to change the name of this variable to something more along of the lines of what the comment above is suggesting? I worry that since `dist` is often used as a name for where built files are output to during a site build that this will be confusing for a future dev.

Reply:
Totally. Done in https://github.com/netlify/cli/pull/5338/commits/d782283cd00af0cb6b0a5cec751ff1c4a901d349.